egs/aspire/s5/local/multi_condition/aspire_prep_rir_noise_list.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
Do you mean you are going to upload the real RIR to openslr.org including the RVB2014, air and RWCP databases?